.ci/setup_env_fedora.sh

```diff
@@ -77,6 +71,20 @@ main()
 		echo "Install ${KATA_KSM_THROTTLER_JOB}"
 		chronic sudo -E dnf -y install ${KATA_KSM_THROTTLER_JOB}
 	fi
+
+	if [ "${TEST_CGROUPSV2}" == "true" ]; then
+		echo "Install podman dependencies"
+		dnf deplist podman --archlist x86_64,noarch | awk '/provider:/ {print $2}' | sort -u | xargs sudo -E dnf -y install
```

Comment:
Can you remove this architecture-specific option?

In fact I think you can do this and simplify it in one go:

```sh
sudo dnf -y builddep podman
```

Reply:
Notice that you are installing the build dependencies for a version that does not correspond to the one you gonna build. Don't know how likely this can be a problem, though.

Using `builddep` seems better but with the cost of adding extra-dependencies, i.e., those needed to build the rpm. Well, on CI it should not be a problem.

Reply:
any idea @wainersm if I need to add something else?

Reply:
it seems that `conmon` needs to be installed before running this. We install `conmon` as part of of the [crio installation](https://github.com/kata-containers/tests/blob/master/.ci/install_crio.sh#L33). maybe we should take the conmon installation out of that script and have an `install_conmon.sh` ? so that we can reuse it here? 
We may also want to build/install podman using its own script (e.g. `install_podman.sh`) as we do for the other pod/container managers.

Reply:
@chavafg maybe we just need to install the package, we do not need to build it as it says at
https://podman.io/getting-started/installation

Reply:
@chavafg the installation will apply only for podman

Reply:
what I do not understand is why
```
dnf deplist podman --archlist x86_64,noarch | awk '/provider:/ {print $2}' | sort -u | xargs sudo -E dnf -y install
```
did the installation of all dependencies (including conmon)

Reply:
> @chavafg maybe we just need to install the package, we do not need to build it as it says at

I would prefer to build it as we have had issues with some versions of `conmon` in the past, so we currently stick to the one in https://github.com/kata-containers/runtime/blob/master/versions.yaml#L183 
you can see related: https://github.com/kata-containers/tests/pull/2171

Reply:
@chavafg , I think that we should use the dnf installation of the packages like I put in my previous comment, the issue is that even if I build or install conmon then I will have issues like
```
Error: could not get runtime: default OCI runtime "crun" not found: invalid argument
```

Reply:
@jodh-intel and @wainersm, I change the command to the original form, the issue is that it does not install all the dependencies that we need (like conmon, crun, etc) and even if I installed them manually, then I got errors like `/usr/share/containers/libpod.conf: No such file or directory`